loader/loader_test.go

```diff
@@ -2558,3 +2559,46 @@ services:
 		},
 	})
 }
+
+type customLoader struct {
+	prefix string
+}
+
+func (c customLoader) Accept(s string) bool {
+	return strings.HasPrefix(s, c.prefix+":")
+}
+
+func (c customLoader) Load(ctx context.Context, s string) (string, error) {
+	path := filepath.Join("testdata", c.prefix, "compose.yaml")
+	_, err := os.Stat(path)
+	return path, err
+}
+
+func TestLoadWithRemoteResources(t *testing.T) {
```

Comment:
I think we could probably use a few additional test cases, e.g. when remote loader returns an error, when you load a remote file that loads ANOTHER remote file, cycle detection with remote resources 😬